### ISSUE TITLE
Section 6.6 "General Introduction " is appropriate to be placed as the first section under chapter 6

### ISSUE THREAD

**Issue by Vidya313** [mschermann/data_viz_reader#1302]
https://github.com/mschermann/data_viz_reader/blob/670e2582620adbc83086a5b018abdeae1293685d/06-process.Rmd#L150

The general introduction addresses the life-cycle of a data analysis project where data visualization is one of the different stages in the process. Hence it makes more sense to have this section first under the chapter 6 and then have all the topics specific to data visualization.

**Comment by AayushiVAgrawal** [comment]
I agree. Since it speaks about a general introduction, it should come in the beginning of the chapter.
Great observation Vidya!

**Comment by rlwy** [comment]
I agree with this too. I wanted to move it while reading the book. It gives the reader a better flow.

**Comment by Psharma2193** [comment]
Yes, I noticed this too. It should be in the  beginning of the section.

**Comment by bharatimalik** [comment]
Yes, I noticed it too. It should definitely be at the beginning of the section.
